src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -94,33 +88,100 @@ protected virtual IEnumerable<FieldType> ProvideFields()
         /// Retuns a list of query arguments within a <see cref="QueryArguments"/> instance, and a
         /// <see cref="IFieldResolver"/> instance, for the specified field.
         /// </summary>
-        protected (QueryArguments? QueryArguments, IFieldResolver Resolver) BuildMethodResolver(MethodInfo methodInfo, FieldType fieldType)
+        protected (QueryArguments? QueryArguments, IFieldResolver Resolver) BuildFieldResolver(MemberInfo memberInfo, FieldType fieldType)
```

Comment:
Do you mean _Field_ here as a _GraphQL field_ and not a _CLR field_ ?

Reply:
Yes. Now this method is used for any member. I thought of BuildMemberResolver or BuildResolver also. What do you think?

Reply:
Notice this also returns query arguments and a resolver, which is immediately set on the `FieldType`.  We could simply make this a void method that sets the arguments and resolver directly on the field.  Since both properties are read/write, a caller still has the opportunity to change the arguments or resolver on the field after calling this method.  In this case the method might be called something different.  Perhaps the order of the method arguments would be swapped.  For example:

```cs
public void BuildFieldType(FieldType fieldType, MemberInfo memberInfo)
```

Reply:
See:
- #2939

Reply:
OK for `protected void BuildFieldType(FieldType fieldType, MemberInfo memberInfo)`, maybe even `ConfigureFieldType`